packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.routing.js

```diff
@@ -0,0 +1,58 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.xdsl-meetings', {
+    url: '/xdsl/:serviceName/meetings',
+    views: {
+      'telecomView@telecom': 'packXdslMeetings',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      loadMeetings: /* @ngInject */ ($rootScope, OvhApiXdsl, serviceName) => {
+        return OvhApiXdsl.v6().searchOrderMeetings($rootScope, {
+          serviceName,
+        });
+      },
+      slots: /* @ngInject */ ($translate, loadMeetings) => {
+        const meetingSlots = {};
+        const meetings = [];
+        let showMeetingSlots = false;
+
+        const { result } = loadMeetings;
```

Comment:
looks like `loadMeetings` is a promise no ?

Reply:
yes and ?

Reply:
No it's an object and it works with the correct result.
If I write :
```
return OvhApiXdsl.v6().searchOrderMeetings($rootScope, {
          serviceName,
        }).$promise
```
I have the Oops page.

Reply:
`loadMeetings` returns the result of `OvhApiXdsl.v6().searchOrderMeetings` which is a `Poller` (https://github.com/ovh-ux/ovh-api-services/blob/master/src/api/xdsl/xdsl.v6.service.js#L94-L117)